ui/components/app/assets/asset-list/sort-control/index.scss

```diff
@@ -0,0 +1,30 @@
+.selectable-list-item-wrapper {
+  position: relative;
+}
+
+.selectable-list-item {
+  cursor: pointer;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 16px;
+  padding-bottom: 16px;
```

Comment:
We can just do `padding: 16px;`

Reply:
Good call. I should probably use the design system props here actually

Reply:
https://github.com/MetaMask/metamask-extension/pull/27184/commits/943ee72a2103c06b605ec6441b464ca8824dc845